zerver/views/users.py

```diff
@@ -144,7 +145,11 @@ def deactivate_user_own_backend(request: HttpRequest, user_profile: UserProfile)
     if user_profile.is_realm_owner and check_last_owner(user_profile):
         raise CannotDeactivateLastUserError(is_last_owner=True)
 
-    do_deactivate_user(user_profile, acting_user=user_profile)
+    group_setting_updates = check_group_permission_updates_for_deactivating_user(user_profile)
```

Comment:
We have error handling for this function everywhere. Is there some error handling we need to do in views as well?

Reply:
For other places, we cannot raise a `JsonableError` directly, like for managements command we raise `CommandError` and for support views we return a success response with context containing error message.